cmd/ddev/cmd/config.go

```diff
@@ -35,6 +52,12 @@ var ConfigCommand = &cobra.Command{
 		err = c.Write()
 		if err != nil {
 			util.Failed("Could not write ddev config file: %v\n", err)
+
+		}
+
+		// If a provider is specified, prompt about whether to do an import after config.
+		if provider != "" {
+			util.Success("Configuration complete. You may now run `ddev start` or `ddev pull`")
```

Comment:
I'm going to make these changes but it feels like it may be something worth discussing at some point. I agree that this is not markdown and we don't get improved formatting from this, but in many places (markdown, slack chats, shell, gitter chats) backticks are used to indicate commands or shell output.

I can accept if I'm the only one, but to me at least, seeing backticks around text makes it much more clear that it's an intended command than quotes do.